google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -702,6 +722,29 @@ def key(self):
         return self['name'].split('/', 1)[-1]
 
 
+class ResourceManagerAccessPolicy(resource_class_factory('crm_access_policy',
+                                                         None)):
+    """The Resource implementation for Resource Manager Access Policy."""
+
+    def key(self):
```

Comment:
Any reason why you cant use the following?

```
class ResourceManagerAccessPolicy(resource_class_factory('crm_access_policy',
                                                         'name')):
```

Reply:
`name` for access_policy is not unique. For access_policy it is `//cloudresourcemanager.googleapis.com/organizations/111222333`. Hence `name` cannot be used. Just like `ResourceManagerOrgPolicy()`, for `ResourceManagerAccessPolicy()` we need to use an empty string or `None`.

Reply:
The key method you implemented, is to return the name field. I don't see how that is different than passing in 'name' as the 2nd parameter.

Reply:
Updated